Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.py

```diff
@@ -990,14 +1041,21 @@ def alert_update_state(client, data_args) -> Tuple[str, dict, Union[list, dict]]
         :rtype: ``tuple``
 
     """
-    alert_ids = argToList(data_args.get('alert_ids'))
+    alert_ids = [int(alert) for alert in argToList(data_args.get('alert_ids'))]
     state = data_args.get('state')
 
     body = {
-        'state': state.lower(),
-        'id': alert_ids
+        'state': state.lower()
     }
-    client.do_request('PUT', '/plugin/products/detect3/api/v1/alerts/', data=body)
+    if client.api_version == "4.x":
+        if len(alert_ids) == 1:
+            client.do_request('PUT', f'/plugin/products/threat-response/api/v1/alerts/{str(alert_ids[0])}', data=body)
+        else:
+            client.do_request('PUT', '/plugin/products/threat-response/api/v1/alerts/',
+                              data=body, params={'id': alert_ids})
```

Comment:
Do we have to split the endpoints?
Won't this work with 1 alert_id?

Reply:
no it does not.